pom.xml

```diff
@@ -117,10 +117,6 @@
             <artifactId>jjwt-jackson</artifactId>
             <version>${jjwt.version}</version>
         </dependency>
-        <dependency>
-            <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-docker-compose</artifactId>
-        </dependency>
```

Comment:
still exists.

Reply:
i have some problems with ci build so i decided to remove this dependency thinking it will help. But problem was not here and i forgot to add it back and push, in the last commit i add it back)